cdk/README.md

```diff
@@ -1,30 +1,50 @@
-# CDK L1 Dev and Publishing
+# MongoDB Atlas AWS CDK Resources
+Use MongoDB Atlas AWS CDK constructs to manage [MongoDB Atlas](https://www.mongodb.com/cloud/atlas).
```

Comment:
@Zuhairahmed I'm not sure if there's more you'd like us to say here - there isn't an existing general introduction to our cdk integration as it was all combined with cfn

Reply:
sure added a few more sentences to help introduce tool to users